README.md

```diff
@@ -18,128 +18,177 @@ your part.
-If your boot chain of trust includes linux kernel, is
-"efi: Restrict efivar_ssdt_load when the kernel is locked down"
-upstream commit 1957a85b0032a81e6482ca4aab883643b8dae06e applied ?
-Is "ACPI: configfs: Disallow loading ACPI tables when locked down"
-upstream commit 75b0cea7bf307f362057cc778efe89af4c615354 applied ?
+### How do you manage and protect the keys used in your SHIM?
 -------------------------------------------------------------------------------
-[your text here]
+`[your text here]`
 
 -------------------------------------------------------------------------------
-If you use vendor_db functionality of providing multiple certificates and/or
-hashes please briefly describe your certificate setup. If there are allow-listed hashes
-please provide exact binaries for which hashes are created via file sharing service,
-available in public with anonymous access for verification
+### Do you use EV certificates as embedded certificates in the SHIM?
 -------------------------------------------------------------------------------
-[your text here]
+`[your text here]`
 
 -------------------------------------------------------------------------------
-If you are re-using a previously used (CA) certificate, you will need
-to add the hashes of the previous GRUB2 binaries to vendor_dbx in shim
-in order to prevent GRUB2 from being able to chainload those older GRUB2
-binaries. If you are changing to a new (CA) certificate, this does not
-apply. Please describe your strategy.
+### If you use new vendor_db functionality, are any hashes allow-listed?
+### If yes: for what binaries?
 -------------------------------------------------------------------------------
-[your text here]
+`[your text here]`
 
 -------------------------------------------------------------------------------
-What OS and toolchain must we use to reproduce this build?  Include where to find it, etc.  We're going to try to reproduce your build as close as possible to verify that it's really a build of the source tree you tell us it is, so these need to be fairly thorough. At the very least include the specific versions of gcc, binutils, and gnu-efi which were used, and where to find those binaries.
-If the shim binaries can't be reproduced using the provided Dockerfile, please explain why that's the case and the differences would be.
+### If your boot chain of trust includes a Linux kernel:
+### Is upstream commit [1957a85b0032a81e6482ca4aab883643b8dae06e "efi: Restrict efivar_ssdt_load when the kernel is locked down"](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=1957a85b0032a81e6482ca4aab883643b8dae06e) applied?
+### Is upstream commit [75b0cea7bf307f362057cc778efe89af4c615354 "ACPI: configfs: Disallow loading ACPI tables when locked down"](https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=75b0cea7bf307f362057cc778efe89af4c615354) applied?
```

Comment:
When this question was introduced it only asked for one commit in the ISSUE_TEMPLATE, but two commits in the README. I don't know why, so I've put both here. That might be wrong.